database/templates/openshift.backups.yml

```diff
@@ -0,0 +1,162 @@
+# https://github.com/BCDevOps/backups/blob/master/openshift/templates/backup-cronjob/backup-cronjob.yaml
+---
+kind: "Template"
+apiVersion: template.openshift.io/v1
+parameters:
+  - name: NAME
+    description: Module name
+    required: true
+  - name: ZONE
+    description: Deployment zone, e.g. pr-### or prod
+    required: true
+  - name: COMPONENT
+    description: Component name
+    value: database-backup
+  - name: "DATABASE_DEFAULT_PORT"
+    description: "The configured port for the database service"
+    value: "5432"
+  - name: "BACKUP_STRATEGY"
+    description: "The strategy to use for backups; for example daily, or rolling."
+    required: true
+    value: "rolling"
+  - name: "BACKUP_DIR"
+    description: "The name of the root backup directory"
+    required: true
+    value: "/backups/"
+  - name: "NUM_BACKUPS"
+    description: "The number of backup files to be retained.  Used for the `daily` backup strategy.  Ignored when using the `rolling` backup strategy."
+    required: false
+    value: "5"
+  - name: "DAILY_BACKUPS"
+    description: "The number of daily backup files to be retained.  Used for the `rolling` backup strategy."
+    required: false
+    value: "7"
+  - name: "WEEKLY_BACKUPS"
+    description: "The number of weekly backup files to be retained.  Used for the `rolling` backup strategy."
+    required: false
+    value: "4"
+  - name: "MONTHLY_BACKUPS"
+    displayName: "Number of Monthly Backups to Retain"
+    description: "The number of monthly backup files to be retained.  Used for the `rolling` backup strategy."
+    required: false
+    value: "1"
+  - name: "JOB_SERVICE_ACCOUNT"
+    description: "Name of the Service Account To Exeucte the Job As."
+    value: "default"
+    required: true
+  - name: "SUCCESS_JOBS_HISTORY_LIMIT"
+    description: "The number of successful jobs that will be retained"
+    value: "5"
+    required: true
+  - name: "FAILED_JOBS_HISTORY_LIMIT"
+    description: "The number of failed jobs that will be retained"
+    value: "2"
+    required: true
+  - name: "JOB_BACKOFF_LIMIT"
+    description: "The number of attempts to try for a successful job outcome"
+    value: "0"
+  - description: Volume space available for data, e.g. 512Mi, 2Gi.
+    name: PVC_SIZE
+    value: 1Gi
```

Comment:
same for volume here.

Reply:
Thanks for the improvements!  Changes made.  :)